debian_db_packages.lst

```diff
@@ -0,0 +1 @@
+postgresql
```

Comment:
Good question. It looks like Redis was not installed or configured in the Vagrantfile from @omarkhan . All of the tests pass, so there must also be no test for Redis.